src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/SwitchesSpec.groovy

```diff
@@ -415,4 +420,41 @@ class SwitchesSpec extends HealthCheckSpecification {
         cleanup:
         northboundV2.partialSwitchUpdate(sw.dpId, new SwitchPatchDto().tap { it.pop = initConf.pop ?: "" })
     }
+
+    @Tags([VIRTUAL, LOW_PRIORITY])
+    def "System is able to fix rule and meter related to this rule by synchronize switch operation"() {
+        /**
+         * It works for default meters only.
+         * For example some default rule is missing,
+         * then syncSwitch before installing this rule, will check related meter and fix it (if needed).
+         */
+        given: "An active switch with missing default rule and misconfigured meter related to the rule"
+        def sw = topology.activeSwitches.first()
+        northbound.deleteSwitchRules(sw.dpId, DeleteRulesAction.REMOVE_BROADCAST)
+        def meterIdBroadcastCookie = MeterId.createMeterIdForDefaultRule(VERIFICATION_BROADCAST_RULE_COOKIE).getValue()
+        def meterToManipulate = northbound.getAllMeters(sw.dpId).meterEntries
+                .find{ it.meterId == meterIdBroadcastCookie }
+        def newBurstSize = meterToManipulate.burstSize + 100
+
+        lockKeeper.updateBurstSizeAndRate(sw, meterToManipulate.meterId, newBurstSize, meterToManipulate.rate)
+        Wrappers.wait(RULES_DELETION_TIME) {
+            def validateInfo = northbound.validateSwitch(sw.dpId)
+            assert validateInfo.rules.missing.size() == 1
+            assert validateInfo.meters.misconfigured.size() == 1
+            assert validateInfo.meters.misconfigured[0].actual.burstSize == newBurstSize
+            assert validateInfo.meters.misconfigured[0].expected.burstSize == meterToManipulate.burstSize
+        }
+
+        when: "Sycnhronize switch"
+        northbound.synchronizeSwitch(sw.dpId, false)
+
+        then: "The misconfigured meter was fixed an moved to the 'proper' section"
```

Comment:
Please mention the 'rule' as well, in order to match what is described in test name and 'given' block